lib/model/emoji.dart

```diff
@@ -54,9 +61,36 @@ class ImageEmojiDisplay extends EmojiDisplay {
 /// We do this based on a user preference,
 /// and as a fallback when the Unicode or image approaches fail.
 class TextEmojiDisplay extends EmojiDisplay {
-  TextEmojiDisplay({required super.emojiName});
+  const TextEmojiDisplay({required super.emojiName});
 }
 
+const popularUnicodeEmojis = [
+  UnicodeEmojiDisplay(
```

Comment:
Yeah, it looks like basically this one spot needs to switch to EmojiCandidate, and then that'll supply an `emojiCode` at the one place that needs it, in ReactionButtons.

EmojiCandidate is the right class here conceptually too. In principle, and in the future, the emoji we show in the quick list at the top of the action sheet should depend on actual usage and may include realm emoji (or perhaps even sometimes the Zulip extra emoji, `:zulip:`).